browser/src/map/Clipboard.js

```diff
@@ -1351,7 +1362,19 @@ L.Clipboard = L.Class.extend({
 		this._map.uiManager.showYesNoButton(id + '-box', /*title=*/'', /*message=*/'', /*yesButtonText=*/_('Paste from this document'), /*noButtonText=*/_('Cancel paste special'), /*yesFunction=*/function() {
 			app.socket.sendMessage('uno .uno:PasteSpecial');
 		}, /*noFunction=*/null, /*cancellable=*/true);
+
+		this._openPasteSpecialPopupImpl(id);
+	},
+
+	_openPasteSpecialPopupImpl: function (id) {
 		var box = document.getElementById(id + '-box');
+
+		// TODO: do it JSDialog native...
+		if (!box) {
+			setTimeout(() => { this._openPasteSpecialPopupImpl(id) }, 10);
+			return;
+		}
+
```

Comment:
Same here. Note that also using a layouting task would also mean we don't need to add an extra class method, it could just be a lambda.